.github/workflows/release_gem.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: Release
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    # Pattern matched against refs/tags
+    tags: ['**']
+
+jobs:
+  release:
+    name: Release gem
+    uses: theforeman/actions/.github/workflows/release-gem.yml@v0
+    with:
+      allowed_owner: ATIX-AG
```

Comment:
Should be 'theforeman'. See also https://github.com/theforeman/foreman_ansible/blob/master/.github/workflows/release_gem.yml#L5

Reply:
Done, thanks!